src/components/plots/styling/PlotStyling.jsx

```diff
@@ -19,8 +19,6 @@ import ColourInversion from './ColourInversion';
 import ExpressionValuesType from './ExpressionValuesType';
 import ExpressionValuesCapping from './ExpressionValuesCapping';
 
-import MetadataTracksDisplayEditor from './MetadataTracksDisplayEditor';
```

Comment:
why is this no longer used?

Reply:
Duplicate component - look at `HeatmapMetadataTrackSettings`